objectRecognitionSource/README.md

```diff
@@ -185,6 +187,12 @@ the previous is sent.
     *   (**NOTE:** Previously named "pollRate". For a limited amount of time, both "pollTime" and "pollRate"
     will be valid General Config options, but in the future only "pollTime" will be supported.)
 *   allowQueries: This option allows Queries to be received when set to `true'
+*   maxRunningThreads: Optional. Only used if `pollTime` has been specified. The maximum number of threads running at any 
+given point for polling requests from the specific VANTIQ source. Must be a positive integer. Default value is 10.
+*   maxQueuedTask: Optional. Only used if `pollTime` has been specified. The maximum number of queued tasks at any given point 
+for polling requests from the specific VANTIQ source. Must be a positive integer. Default value is 20.
+    *   **NOTE:** The default behavior of the Object Recognition Source is to process the captured frames in parallel. If you 
+    would like the source to process frames sequentially, then `maxRunningThreads` and `maxQueuedTasks` must both be set to 1.
```

Comment:
Have we tried this?  That is, I suspect what happens here is that we process one image.  If another comes along, it will fail to queue (if we're still processing) and add some error but drop that image.

Or, is what will happen that we continually replace the queued task until the first one finishes, at which point we'll process the "latest" queued task (due to discardOld policy)?

Reply:
The behavior is the latter.

We will set up a queue with a size of one, and allow one thread to run at a time. Assuming the pollTime is faster than the time to execute a task, the queue will continually replace old tasks with the newest poll request until the original task as finished executing.

So in effect, the frames are being processed one at a time, but it is not true "sequential" processing since they are still processed in their own independent thread.